### ISSUE TITLE
Cannot convert model for DSP

### ISSUE THREAD

**Issue by mexeniz** [XiaoMi/mace#376]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 18.04
- **NDK version(e.g., 15c)**: 17c
- **GCC version(if compiling for host, e.g., 5.4.0)**: 7.3.0
- **MACE version (Use the command: git describe --long --tags)**: v0.10.0-88-g3beb766d
- **Python version(2.7)**: 3.6.7
- **Bazel version (e.g., 0.13.0)**: 0.13.0

### Model deploy file (mobilenet-dsp.yml)
```yaml
library_name: mobilenet
target_abis: [armeabi-v7a]
#target_socs: [sdm845]
model_graph_format: code
model_data_format: code
models:
  mobilenet_v2:
    platform: tensorflow
    model_file_path: https://cnbj1.fds.api.xiaomi.com/mace/miai-models/mobilenet-v2/mobilenet-v2-1.0.pb
    model_sha256_checksum: 369f9a5f38f3c15b4311c1c84c032ce868da9f371b5f78c13d3ea3c537389bb4
    subgraphs:
      - input_tensors:
          - input
        input_shapes:
          - 1,224,224,3
        output_tensors:
          - MobilenetV2/Predictions/Reshape_1
        output_shapes:
          - 1,1001
    # runtime: cpu+gpu
    runtime: dsp
    limit_opencl_kernel_time: 0
    obfuscate: 0
    winograd: 0
```

### Describe the problem
I failed to convert the model for DSP. My yaml config bases on mace/examples/android/mobilenet.yml. How can I fix this DSP converter? 

### To Reproduce
Steps to reproduce the problem:
```bash
1. cd mace
2. python tools/converter.py convert --config=mace/examples/android/mobilenet-dsp.yml
```

### Error information / logs
Please include the **full** log and/or traceback here.
```bash
~/mace$ python tools/converter.py convert --config=mace/examples/android/mobilenet-dsp.yml
--------------------------------------
         Common Configuration         
--------------------------------------
|        key        |     value      |
======================================
|       library_name|       mobilenet|
--------------------------------------
|        target_abis| ['armeabi-v7a']|
--------------------------------------
|        target_socs|              []|
--------------------------------------
| model_graph_format|            code|
--------------------------------------
|  model_data_format|            code|
--------------------------------------

* Generate mace engine creator source
Generate mace engine creator source done!

**********************************************
          Convert mobilenet_v2 model          
**********************************************

INFO: Analysed target //mace/python/tools:converter (0 packages loaded).
INFO: Found 1 target...
Target //mace/python/tools:converter up-to-date:
  bazel-bin/mace/python/tools/converter
INFO: Elapsed time: 0.181s, Critical Path: 0.00s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
....
MobilenetV2/expanded_conv_14/add (Eltwise, index:53): [[1, 7, 7, 160]]
MobilenetV2/expanded_conv_15/expand/Relu6 (Conv2D, index:54): [[1, 7, 7, 960]]
MobilenetV2/expanded_conv_15/depthwise/Relu6 (DepthwiseConv2d, index:55): [[1, 7, 7, 960]]
MobilenetV2/expanded_conv_15/project/BatchNorm/FusedBatchNorm (Conv2D, index:56): [[1, 7, 7, 160]]
MobilenetV2/expanded_conv_15/add (Eltwise, index:57): [[1, 7, 7, 160]]
MobilenetV2/expanded_conv_16/expand/Relu6 (Conv2D, index:58): [[1, 7, 7, 960]]
MobilenetV2/expanded_conv_16/depthwise/Relu6 (DepthwiseConv2d, index:59): [[1, 7, 7, 960]]
MobilenetV2/expanded_conv_16/project/BatchNorm/FusedBatchNorm (Conv2D, index:60): [[1, 7, 7, 320]]
MobilenetV2/Conv_1/Relu6 (Conv2D, index:61): [[1, 7, 7, 1280]]
MobilenetV2/Logits/AvgPool (Pooling, index:62): [[1, 1, 1, 1280]]
MobilenetV2/Logits/Conv2d_1c_1x1/BiasAdd (Conv2D, index:63): [[1, 1, 1, 1001]]
MobilenetV2/Logits/Squeeze (Squeeze, index:64): [[1, 1001]]
MobilenetV2/Predictions/Softmax (Softmax, index:65): [[1, 1001]]
Convert mace graph to hexagon.
Traceback (most recent call last):
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter.py", line 363, in <module>
    main(unused_args=[sys.argv[0]] + unparsed)
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter.py", line 203, in main
    output_graph_def = converter.run()
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter_tool/hexagon_converter.py", line 125, in run
    self.convert_ops()
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter_tool/hexagon_converter.py", line 171, in convert_ops
    self.add_min_max_const_node(op, op.input[0])
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter_tool/hexagon_converter.py", line 358, in add_min_max_const_node
    raise Exception('Quantize info not found: ', tensor_name)
Exception: ('Quantize info not found: ', 'input:0')
Traceback (most recent call last):
  File "tools/converter.py", line 1300, in <module>
    flags.func(flags)
  File "tools/converter.py", line 774, in convert_func
    convert_model(configs, flags.cl_mem_type)
  File "tools/converter.py", line 704, in convert_model
    ",".join(model_config.get(YAMLKeyword.graph_optimize_options, [])))
  File "/home/pawissakan/mace/tools/sh_commands.py", line 532, in gen_model_code
    _fg=True)
  File "/home/pawissakan/.pyenv/versions/mace/lib/python3.6/site-packages/sh.py", line 1413, in __call__
    raise exc
sh.ErrorReturnCode_1: 

  RAN: /home/pawissakan/.pyenv/versions/mace/bin/python bazel-bin/mace/python/tools/converter -u --platform=tensorflow --model_file=builds/downloads/d4e627ce1b15ed6f96aefffe315a8dc8.pb --weight_file= --model_checksum=369f9a5f38f3c15b4311c1c84c032ce868da9f371b5f78c13d3ea3c537389bb4 --weight_checksum= --input_node=input --input_data_formats=NHWC --output_node=MobilenetV2/Predictions/Reshape_1 --output_data_formats=NHWC --check_node= --runtime=dsp --template=mace/python/tools --model_tag=mobilenet_v2 --input_shape=1,224,224,3 --input_range= --output_shape=1,1001 --check_shape= --dsp_mode=0 --embed_model_data=True --winograd=0 --quantize=0 --quantize_range_file= --change_concat_ranges=0 --obfuscate=0 --output_dir=mace/codegen/models/mobilenet_v2 --model_graph_format=code --data_type=uint8 --graph_optimize_options= --cl_mem_type=image

  STDOUT:


  STDERR:


```

**Comment by lee-bin** [comment]
DSP can only be used for quantized models.
You can use this model as an example from [mace-models](https://github.com/XiaoMi/mace-models):
https://github.com/XiaoMi/mace-models/blob/master/mobilenet-v1/mobilenet-v1-quantize-retrain-dsp.yml

**Comment by mexeniz** [comment]
@lee-bin It works, thanks!!
I also use change from Python3 to Python2, because there is a line of code that has 'long' variable.
'long' is removed from Python3.

**Comment by yejw5** [comment]
@mexeniz Could you afford which line is? We'll fix it.

**Comment by mexeniz** [comment]
@yejw5 In latest commit: 2a56318518b8aacb15d2d1e522d3f630cbee772f

```
Traceback (most recent call last):
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter.py", line 363, in <module>
    main(unused_args=[sys.argv[0]] + unparsed)
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter.py", line 197, in main
    output_graph_def, quantize_activation_info = mace_transformer.run()
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter_tool/transformer.py", line 128, in run
    changed = transformer()
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter_tool/transformer.py", line 1536, in quantize_weights
    self.quantize_tensor(tensor)
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/converter_tool/transformer.py", line 1511, in quantize_tensor
    tensor.float_data)
  File "/home/pawissakan/mace/bazel-bin/mace/python/tools/converter.runfiles/mace/mace/python/tools/quantization/quantize_util.py", line 135, in quantize_bias_for_hexagon
    output = np.clip((np.round(zero + data / scale).astype(long)),
NameError: name 'long' is not defined
Traceback (most recent call last):
  File "tools/converter.py", line 1308, in <module>
    flags.func(flags)
  File "tools/converter.py", line 782, in convert_func
    convert_model(configs, flags.cl_mem_type)
  File "tools/converter.py", line 712, in convert_model
    ",".join(model_config.get(YAMLKeyword.graph_optimize_options, [])))
  File "/home/pawissakan/mace/tools/sh_commands.py", line 532, in gen_model_code
    _fg=True)
  File "/home/pawissakan/.pyenv/versions/mace/lib/python3.6/site-packages/sh.py", line 1413, in __call__
    raise exc
sh.ErrorReturnCode_1: 

  RAN: /home/pawissakan/.pyenv/versions/mace/bin/python bazel-bin/mace/python/tools/converter -u --platform=tensorflow --model_file=builds/downloads/32cb63fe2fe9eb13f7d77dd1ab7b6298.pb --weight_file= --model_checksum=c3faa9bd790310eb3dec411d2f2e2dd8802394e31ccb47daaa5e3b6369fadfa3 --weight_checksum= --input_node=input --input_data_formats=NHWC --output_node=MobilenetV1/Predictions/Softmax:0 --output_data_formats=NHWC --check_node=MobilenetV1/Logits/Conv2d_1c_1x1/BiasAdd:0 --runtime=dsp --template=mace/python/tools --model_tag=mobilenet_v1_quantize_retrain --input_shape=1,224,224,3 --input_range=-1.0,1.0 --output_shape=1,1001 --check_shape=1,1,1,1001 --dsp_mode=0 --embed_model_data=True --winograd=0 --quantize=1 --quantize_range_file= --change_concat_ranges=0 --obfuscate=0 --output_dir=mace/codegen/models/mobilenet_v1_quantize_retrain --model_graph_format=code --data_type=uint8 --graph_optimize_options= --cl_mem_type=image

  STDOUT:


  STDERR:

```

**Comment by yejw5** [comment]
@mexeniz It's fixed in b6b51e36fb0cbdcba77a799a600292741fcd5c93
